src/GraphBLASExt/GxB_Delete.c

```diff
@@ -1,25 +1,21 @@
 #include "./GxB_Delete.h"
 
-GrB_Info GxB_Matrix_Delete
+void GxB_Matrix_Delete
 (
     GrB_Matrix M,
     GrB_Index row,
     GrB_Index col
 )
 {
-    GrB_Matrix Z ;  // 1X1 empty matrix.
-    GrB_Matrix_new (&Z, GrB_BOOL, 1, 1) ;
-
-    GrB_Info info = GxB_Matrix_subassign (M,
-                                          GrB_NULL,
-                                          GrB_NULL,
-                                          Z,
-                                          &row,
-                                          1,
-                                          &col,
-                                          1,
-                                          GrB_NULL) ;
-
-    GrB_Matrix_free(&Z);
-    return info;
+    GrB_Matrix_setElement_BOOL(M,
+                               false,
+                               row,
+                               col) ;
+    GxB_Matrix_select(M,
+                      GrB_NULL,
+                      GrB_NULL,
+                      GxB_NONZERO,
+                      M,
+                      GrB_NULL,
+                      GrB_NULL) ;
```

Comment:
Guaranteed to apply pending updates, and I found it to be slightly faster than the previous solution.